src/slacko.ml

```diff
@@ -1593,14 +1562,13 @@ let search base token ?sort ?sort_dir ?highlight ?count ?page query_ =
     | #bot_error as res -> res
     | _ -> `Unknown_error
 
-let search_all = search @@ endpoint "search.all"
-let search_files = search @@ endpoint "search.files"
-let search_messages = search @@ endpoint "search.messages"
+let search_all session = search session @@ endpoint session "search.all"
```

Comment:
Could be done nicer with supplying the curried `endpoint "search.all"` and then calling `base session` inside, which removes some duplication.